### ISSUE TITLE
An exception occurred while invoking executor 'executor://xunit/VsTestRunner2': Sequence contains no elements

### ISSUE THREAD

**Issue by exalted** [xunit/xunit#168]
Hi,

when I run all of the tests via `Test > Run > All Tests` I get the following error:

![untitled](https://cloud.githubusercontent.com/assets/314398/3965918/8d2ad618-279c-11e4-9d5e-21461de5cf70.jpg)

Instead, if I select all of the tests from the Test Explorer and `Run Selected Tests` the same error won't appear.

This is Visual Studio Professional 2012, .NET Framework Version 4.5, xUnit.net package via NuGet at Version 1.9.2 and xUnit.net runner for Visual Studio Version 0.99.8.

Any ideas please?

Thank you.

**Comment by RobertBernstein** [comment]
I just ran into this problem, but it was because I had xUnit 2.0.0-alpha-build1611 installed in an old test project and xUnit 2.0.0-rc1-build2826 installed for a newer test project, both in the same solution, both installed via NuGet.  Once I removed all references to the older version of xUnit, the VS Test Runner (xunit.runner.visualstudio 2.0.0-rc1-build1030) started behaving correctly.  I hope this helps save someone else some difficulty.
